### ISSUE TITLE
Double dot (..) in REPL cause segmentation fault

### ISSUE THREAD

**Issue by dram** [trealla-lang/trealla#19]
Double dot (`..`) in REPL will cause segmentation fault, e.g.:

```
% ./tpl 
Trealla v0.1alpha, 64-bits (int=128-bits), Nov 28 2016
?- abc..
Segmentation fault
```

**Comment by infradig** [comment]
This was caused by fix for #16. The fix is in the pipeline...

**Comment by dram** [comment]
OK, no hurry. :)
